app/sections/view-solution-capabilities/template.njk

```diff
@@ -23,38 +23,67 @@
   <h3>Generated HTML</h3>
   <pre><code>
     {{- '
-    <div class="nhsuk-width-container" data-test-id="view-solution-capabilities">
-      <div class="nhsuk-grid-row">
+    <div class="nhsuk-width-container"
+      <div class="nhsuk-width-container" data-test-id="view-solution-capabilities">
         <div class="bc-c-title-block">
           <h3 class="nhsuk-u-margin-bottom-2">Capabilities met</h3>
         </div>
-        <div class="nhsuk-grid-column-two-thirds">
-          <p>
-            description
-          </p>
-          <div>
-            <div data-test-id="view-question-data-bulletlist">
-              <ul>
-                <li>
-                  <label class="nhsuk-label">
+        <div class="nhsuk-grid-row nhsuk-u-padding-left-3">
+          <div class="nhsuk-grid-column-two-thirds">
+            <p>
+              description
+            </p>
+            <div>
+              <div data-test-id="view-question-data-bulletlist">
+                <ul>
+                  <li>
+                    <label class="nhsuk-label">
                     capability 1
-                  </label>
-                </li>
-                <li>
-                  <label class="nhsuk-label">
+                    </label>
+                  </li>
+                  <li>
+                    <label class="nhsuk-label">
                     capability 2
-                  </label>
-                </li>
-                <li>
-                  <label class="nhsuk-label">
+                    </label>
+                  </li>
+                  <li>
+                    <label class="nhsuk-label">
                     capability 3
-                  </label>
-                </li>
-              </ul>
+                    </label>
+                  </li>
+                </ul>
+              </div>
             </div>
           </div>
         </div>
       </div>
+      data-test-id="view-features">
+      <div class="bc-c-title-block">
+        <h3 class="nhsuk-u-margin-bottom-2">Features</h3>
+      </div>
+      <div class="nhsuk-grid-row nhsuk-u-padding-left-3">
+        <div class="nhsuk-grid-column-two-thirds">
+          <div data-test-id="view-question-data-bulletlist">
+            <ul>
+              <li>
+                <label class="nhsuk-label">
+                Feature 1
+                </label>
+              </li>
+              <li>
+                <label class="nhsuk-label">
+                Feature 2
+                </label>
+              </li>
+              <li>
+                <label class="nhsuk-label">
+                Feature 3
+                </label>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>

```

Comment:
I don't think this should belong here `view-features` section should belong here

Reply:
it should, it is capability list